src/Symfony/Component/Stream/FilesystemStreamWrapper.php

```diff
@@ -0,0 +1,188 @@
+<?php
```

Comment:
should there be also `declare(strict_types=1);`

Reply:
don't know yet: #28423 :) also it might have impact here, as we're in a wrapper scope and each global function call might trigger the wrapper.